example/per.jonprl

```diff
@@ -0,0 +1,125 @@
+Operator per-squash : (0).
+Prefix 7 "⇃" := per-squash.
+[⇃ A] =def= [per(lam(_.lam(_.A)))].
```

Comment:
You're right this one is different from the one in our continuity paper.
I'll use another symbol.
On Sep 3, 2015 3:33 PM, "Jonathan Sterling" <notifications@github.com>
wrote:

> In example/per.jonprl
> <https://github.com/jonsterling/JonPRL/pull/227#discussion_r38652078>:
>
> > @@ -0,0 +1,125 @@
> > +Operator per-squash : (0).
> > +Prefix 7 "⇃" := per-squash.
> > +[⇃ A] =def= [per(lam(_.lam(_.A)))].
>
> I thought that the half-downward-arrow was for the "quotient squash",
> which expresses the PER {(M, N) | M in A, N in A}... This one seems
> different, since its inhabitants are anything at all so long as A is
> inhabited; am I reading this right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jonsterling/JonPRL/pull/227/files#r38652078>.
>